tests/remote.rs

```diff
@@ -0,0 +1,409 @@
+#[macro_use]
```

Comment:
I assume this file is mostly just a copy of the tests that used to be in `tests/common.rs`. Any other changes?

Reply:
It was just a plain copy out of  `tests/common`.